templates/data/opensearch/support.html

```diff
@@ -0,0 +1,544 @@
+              for complex projects, our experienced database performance specialists are prepared to assist
+              you with any challenge.
+            </p>
+            <p>
+              Deal with issues faster and guarantee the ongoing performance of your deployments with 24/7/365
+              consultative OpenSearch support.
+            </p>
+          </div>
+          <hr />
+          <a class="p-button--positive"
+             href="/contact-us"
+             aria-controls="data-opensearch-modal">Contact us</a>
+          <a class="p-button"
+             href="https://assets.ubuntu.com/v1/6dc3440a-Charmed%20OpenSearch%20-%20Datasheet%20-%20amended%2004%2006%202024.pdf">Download
+          the datasheet<span class="u-off-screen">Download the Charmed OpenSearch datasheet</span></a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <hr class="is-fixed-width" />
+    <div class="row">
+      <div class="col-3 col-medium-2">
+        <h2 class="p-text--small-caps highlight">Why choose Canonical for OpenSearch support</h2>
+      </div>
+      <div class="col-9 col-medium-4">
+        <ul class="p-list--divided">
+          <li class="p-list__item u-no-padding">
+            <p class="p-heading--2 u-no-padding">Up to 10 years support and security maintenance</p>
+          </li>
+          <li class="p-list__item u-no-padding">
+            <p class="p-heading--2 u-no-padding">Low total cost of ownership</p>
+          </li>
+          <li class="p-list__item u-no-padding">
+            <p class="p-heading--2 u-no-padding">Experts in the full open source stack</p>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section u-fixed-width">
+    <hr />
+    <div class="row--50-50 p-section--shallow">
+      <div class="col">
+        <h2>OpenSearch Support</h2>
+      </div>
+      <div class="col">
+        <p>
+          Canonical offers 24x7x365 support for your OpenSearch application. Keep your production environment
+          bug-free and secure with our experts' help.
+        </p>
+      </div>
+    </div>
+    <div class="p-equal-height-row p-section--shallow">
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item u-hide--medium">
+          <div class="p-image-container is-highlighted u-hide--small">
+            {{ image(url="https://assets.ubuntu.com/v1/2833131c-Rectangle 2.png",
+                        alt="",
+                        width="852",
+                        height="1279",
+                        hi_def=True,
+                        loading="lazy",
+                        attrs={"class": "p-image-container__image"}) | safe
+            }}
+          </div>
+        </div>
+        <div class="p-equal-height-row__item">
+          <hr class="p-rule--highlight u-hide--medium u-hide--small" />
+          <h5 class="p-heading--5">Technical support</h5>
```

Comment:
Please change it to h3 for all of the equal height columns. Your previous title was h2, you shouldn't jump from h2 to h5